keyboards/ghost54/info.json

```diff
@@ -0,0 +1,126 @@
+{
+
+    "manufacturer": "Ghost-Spot",
+    "keyboard_name": "Ghost54",
+    "maintainer": "Ghost-Spot",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "url": "https://github.com/Ghost-Spot/qmk_firmware/tree/ghost54/keyboards/ghost54",
+    "bootloader_instructions": "Press the button in the RPi thru each case!",
+    "tags": ["Split", "Column Staggered", "Ortho", "RP2040", "Kailh Speed Silvers", "MX Spaced", "Encoder", "OLED"],
+    "usb": {
+        "pid": "0x0001",
+        "vid": "0x4753",
+        "device_version": "1.0.0"
+    },
+    "matrix_pins": {
+        "cols": ["GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP21", "GP20", "GP19", "GP18", "GP17"]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP26", "pin_b": "GP22"}
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "GP03",
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP26", "pin_b": "GP22"}
+                ]
+            }
+        },
+        "matrix_pins":{
+            "right": {
+                "cols": ["GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+                "rows": ["GP21", "GP20", "GP19", "GP18", "GP17"]
+            }
```

Comment:
Definition of the right half's matrix pins and encoder pins are required **if** they are different from the left half.
In that case, as both halves use identical matrix pins and encoder pins, could `split.encoder` and `split.matrix_pins` be deleted?